File: Modul_5/Bahasa_Python/PRAK503-2510817120005-NorKhalisyah.py
def maksimal(a, b):
    if a > b:
        return a
    else:
        return b

def minimal(a, b):
    if a < b:
        return a
    else:
        return b

maks = -100000
minim = 100000

bilangan = int(input("Masukkan jumlah: "))
nilai_list = list(map(int, input("Masukkan nilai: ").split()))

batas = 0
while batas < bilangan:
    nilai = nilai_list[batas]
    maks = maksimal(maks, nilai)
    minim = minimal(minim, nilai)
    batas += 1

print("Nilai max:", maks)
print("Nilai min:", minim)

File: Modul_5/Bahasa_C/PRAK503-2510817120005-NorKhalisyah.c
#include <stdio.h> 
int maksimal(int a, int b){ 
    if (a > b)
       return a;
    else 
        return b;

} 
int minimal(int a, int b){ 
    if (a < b)
       return a;
    else 
        return b;
} 
int main(){ 
int batas = 0; 
int maks = -100000; 
int minim = 100000;

printf("masukkan jumlah: ");
int bilangan; 
scanf("%d", &bilangan); 

printf("masukkan nilai: ");
while(batas < bilangan){ 
int nilai; 
scanf("%d", &nilai);
maks = maksimal(maks, nilai); 
minim = minimal(minim, nilai); 
batas++; 
} 

printf("nilai max: ");
printf("%d\n", maks);
printf("nilai min: ");
printf("%d",minim); 
}